dashboard.yaml

```diff
@@ -0,0 +1,55 @@
+# Kubernetes configuration for the GHCrawler dashboard.
+#
+# Depends on these services:
+# - redis
+# - crawler
+#
+# Serves content on port 4000
+#
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: dashboard
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dashboard
+    spec:
+      containers:
+      - name: dashboard
+        image: gcr.io/clearly-defined/ghcrawler-dashboard:latest
```

Comment:
@jeffmcaffer I wonder if it should be published on Docker Hub.

Reply:
Probably.  Someone just needs to publish it there.  

I don't think this gcr.io container is actually world readable right now.

Reply:
Yeah, the question is where. This is the crawler dashboard, not particular to ClearlyDefined. This could go in a Microsoft area in dockerhub or we could make a ghcrawler repo.  @iamwillbar do you have a preference/opinion?

Reply:
Following the discussion with @iamwillbar, Microsoft/ghcrawler-dashboard should preferably be pushed to Docker Hub's Microsoft org.

Reply:
If someone with access could build and push it there, that would be appreciated.

Reply:
Will do. It may take a few days though.

Reply:
@geneh  Nudge?

Reply:
yeah, we have hit a couple of process snags on our side. Working through them now. Sorry for the delay.

Reply:
Acknowledged.  I totally understand process snags.